build_defs/java_opts.bzl

```diff
@@ -20,3 +25,13 @@ def protobuf_java_library(**kwargs):
         javacopts = JAVA_OPTS,
         **kwargs
     )
+
+def protobuf_versioned_java_library(**kwargs):
+    osgi_java_library(
+        javacopts = JAVA_OPTS,
+        bundle_doc_url = BUNDLE_DOC_URL,
+        bundle_license = BUNDLE_LICENSE,
+        bundle_version = PROTOBUF_JAVA_VERSION,
```

Comment:
You could bake these into the osgi_java_library rule as private attributes (https://bazel.build/extending/rules#private_attributes_and_implicit_dependencies)

Reply:
gotcha, that makes sense

Reply:
might be worth adding a comment to that effect

Reply:
Done.